core/src/components/card-header/card-header.ionic.scss

```diff
@@ -0,0 +1,10 @@
+@use "./card-header.common";
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+
+// Ionic Card Header
+// --------------------------------------------------
+
+:host {
+  @include globals.padding(globals.$ionic-space-400);
+  gap: globals.$ionic-space-050;
```

Comment:
I reduced this because it is a bit hidden but Figma has the gap on a nested element set to this:

<img width="310" alt="Screenshot 2024-08-09 at 2 52 37 PM" src="https://github.com/user-attachments/assets/3f10bee6-a646-45ff-9b62-be981d12c9ed">

Here is a comparison:

| Figma | Ionic | Diff |
| ---| ---| ---|
| ![Figma Card](https://github.com/user-attachments/assets/919008f9-eff3-4e8a-ba3d-04d662203e0c) | ![Ionic Card](https://github.com/user-attachments/assets/6f24ac98-e285-4c5c-abfa-ecbfba8c98fd) | ![Diff](https://github.com/user-attachments/assets/0604c282-fd64-45b0-b53f-8bbc0a809863) |